composer.json

```diff
@@ -105,6 +105,26 @@
 			"url": "git@github.com:yardinternet/plugin-owc-signicat-openid.git"
 		}
 	],
+	"_comment": [
```

Comment:
Dit is wel chill! Denk dat iedereen even zijn plasje moet doen op de schrijfwijze maar ik ben akkoord.